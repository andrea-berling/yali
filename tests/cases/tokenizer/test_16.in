(79+58) > 84 != ("Success" != "Failure") != (10 >= 69)
