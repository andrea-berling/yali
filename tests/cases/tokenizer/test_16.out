LEFT_PAREN ( null
NUMBER 79 79.0
PLUS + null
NUMBER 58 58.0
RIGHT_PAREN ) null
GREATER > null
NUMBER 84 84.0
BANG_EQUAL != null
LEFT_PAREN ( null
STRING "Success" Success
BANG_EQUAL != null
STRING "Failure" Failure
RIGHT_PAREN ) null
BANG_EQUAL != null
LEFT_PAREN ( null
NUMBER 10 10.0
GREATER_EQUAL >= null
NUMBER 69 69.0
RIGHT_PAREN ) null
EOF  null
